packages/zoe/test/zoeTestHelpers.js

```diff
@@ -3,6 +3,47 @@
 import { E } from '@agoric/eventual-send';
 
 import '../exported';
+import setMathHelpers from '@agoric/ertp/src/mathHelpers/setMathHelpers';
+import { MathKind, amountMath } from '@agoric/ertp';
+
+import { q } from '@agoric/assert';
+
+export const assertAmountsEqual = (
+  t,
+  amount,
+  expected,
+  mathKind = MathKind.NAT,
+  label = '',
+) => {
+  const brandsEqual = amount.brand === expected.brand;
+  let valuesEqual;
+  switch (mathKind) {
+    case MathKind.NAT:
+      valuesEqual = amount.value === expected.value;
+      break;
+    case MathKind.STRING_SET:
+      valuesEqual = setMathHelpers.doIsEqual(amount.value, expected.value);
+      break;
+    case MathKind.SET:
+      valuesEqual = setMathHelpers.doIsEqual(amount.value, expected.value);
+      break;
+    default:
+      valuesEqual = false;
+  }
+
```

Comment:
Why are we doing this? `amountMath.isEqual(x, y)` will do everything we need to do.

Reply:
This gives more detailed error messages when there isn't a match. If that's not valuable, then we can switch all the tests to use `t.is(amountEqual(amount, expected, message))`.

Reply:
also, `amountEqual()` throws when the brands don't match, which is less helpful in a test.

Reply:
We can still have the detailed error messages without having to pass in the mathKind. Some options:

* Use `amountMath.isEqual(` directly. If we know the brands are the same and amountMath.isEqual is false, then we know the values are different and can use the appropriate error message.

* Use `isSetValue` and `isNatValue` to switch which helpers to use based on the value alone.

Both of these don't need the mathKind to be passed in, so we should definitely remove that.

Reply:
> also, amountMath.isEqual() throws when the brands don't match, which is less helpful in a test.

An easy fix for this is only use `amountMath.isEqual` if you've already checked that the brands match

Reply:
done.  Thanks.